apps/module-tap/contracts/Tap.sol

```diff
@@ -84,91 +92,137 @@ contract Tap is EtherTokenConstant, IsContract, AragonApp {
     }
 
     /**
-     * @notice Update maximum monthly tap increase to `@formatPct(_maxMonthlyTapIncreasePct)`%
-     * @param _maxMonthlyTapIncreasePct New maximum monthly tap increase rate
+     * @notice Update maximum tap increase rate to `@formatPct(_maximumTapIncreaseRate)`% per second
+     * @param _maximumTapIncreaseRate New maximum tap increase rate
     */
-    function updateMaxMonthlyTapIncreasePct(uint256 _maxMonthlyTapIncreasePct) external auth(UPDATE_MONTHLY_TAP_INCREASE_ROLE) {
-        require(_maxMonthlyTapIncreasePct < PCT_BASE, ERROR_TAP_INCREASE_PCT_TOO_HIGH);
-
-        _updatemaxMonthlyTapIncreasePct(_maxMonthlyTapIncreasePct);
+    function updateMaximumTapIncreaseRate(uint256 _maximumTapIncreaseRate) external auth(UPDATE_MAXIMUM_TAP_INCREASE_RATE_ROLE) {
+        _updateMaximumTapIncreaseRate(_maximumTapIncreaseRate);
```

Comment:
note: this needs to be way below 100% per second (PCT_BASE) or otherwise will skyrocket and should probably be enforced.